rust_snuba/Cargo.toml

```diff
@@ -23,4 +24,5 @@ env_logger = "0.10.0"
 serde = { version = "1.0", features = ["derive"]}
 serde_json = { version = "1.0" }
 glob = "0.3.1"
-pyo3 = { version = "0.18.1", features = ["auto-initialize"] }
+pyo3 = { version = "0.18.1", features = ["chrono", "extension-module"] }
+clickhouse = "0.11.3"
```

Comment:
Is this needed? Afaict it's not used. There's a clickhouse client example in rust_arroyo already that doesn't use a library.

Reply:
removed